graph/resolver/participant.resolvers.go

```diff
@@ -170,6 +171,16 @@ func (r *participantResolver) AnonDisplayName(ctx context.Context, obj *model.Pa
 	return &fullDisplayName, nil
 }
 
+func (r *participantResolver) MutedForSeconds(ctx context.Context, obj *model.Participant) (*int, error) {
+	var result int
+
+	result = 0
+	if obj.MutedUntil != nil {
+		result = int(obj.MutedUntil.Sub(time.Now()).Seconds())
```

Comment:
nit: You can actually use `.Until` function here (it's syntactic sugar for what you have here)